### PR TITLE
Fix #4030 radia replace uuids

### LINKED ISSUE #4030: radia: replace uuid for object ids
Do this on client and server side. This is to simplify the code and reduce computation, since these ids don't need to be globally unique or cryptographically strong.  I will also better document their purpose.

### PR DESCRIPTION
**Pull request by mkeilman**
Notes:

- I left the "UUID" field handling in place in ```SimDataBase.model_defaults```  since it could be useful, but am not opposed to ditching it
- ctrl-click of objects was not working in the radia viewer - fixed since this is a primary use of the id mapping

### CODE REVIEW

File: sirepo/package_data/static/js/sirepo-components.js [hunk 1]
```diff
@@ -4599,6 +4599,16 @@ SIREPO.app.service('utilities', function($window, $interval) {
         };
     };
 
+    // create a non-cryptographic baseN string, with N <= 64
+    this.randomString = function(base=62, length=32) {
```
Comment (e-carlin) [inline sirepo/package_data/static/js/sirepo-components.js:4603]: YAGNI on specifying the base? Always use base62

File: sirepo/package_data/static/js/sirepo-components.js [hunk 2]
```diff
@@ -4599,6 +4599,16 @@ SIREPO.app.service('utilities', function($window, $interval) {
...
+        const BASE64 = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz+/';
+        const s = BASE64.substring(0, base);
+        return new Array(length)
+            .fill('')
```
Comment (e-carlin) [inline sirepo/package_data/static/js/sirepo-components.js:4607]: Geez. I didn't know you had to do this. JS can be so strange.

**Review by e-carlin — APPROVED** [review]
LGTM one nit.

File: sirepo/package_data/static/js/sirepo-components.js (hunk 1 above)
Comment (mkeilman) [reply sirepo/package_data/static/js/sirepo-components.js]: a4dd0cf

**Pull request merged**